centinel/command.py

```diff
@@ -84,4 +94,4 @@ def _invoke_cmd(self):
             if not line:
                 break
             self.output_callback(self, line, self.process.terminate)
-            self.notifications += line
+            self.notifications += line + "\n"
```

Comment:
are you sure that this doesn't already include newlines? I thought subprocess typically piped the newlines from stdout too.